5-Loops/README.md

```diff
@@ -1,79 +1,37 @@
 # Conditionals
-This project covers conditionals using the IF statement.
-For all the if statements refer to [the documentation](https://cmake.org/cmake/help/latest/command/if.html)
+This project covers loops using the FOREACH and WHILE statement.
+For the full documentation on foreach refer to [foreach](https://cmake.org/cmake/help/latest/command/foreach.html)
 
-# If in general
-The IF statement is defind as follows:
+# FOREACH in general
+The FOREACH statement is defind as follows:
 ```CMake
-if(condition)
+foreach(loop_varialble items)
     do_something()
-endif()
+endforeach()
 ```
 
-A lot of extra expressions can be added to the IF statement, but it is easier to read the original documentation in this case.
+A lot of extra expressions can be added to the FOREACH statement, but it is easier to read the original documentation in this case.
 
-# Logical operators
-You can use logical operators like `AND` or `OR` in front of a variable or boolean expression.
-Conditions can be enclosed in parentheses to create a logic group.
-
-```CMake
-if(something OR (NOT inverted_something AND (another_condition OR last_condition)))
-    do_something_very_special()
-endif()
-```
-
-# Useful operators
-```CMake
-if(COMMAND some_command)
-    # True if some_command is a callable command
-endif()
-```
+# FOREACH operators
 ```CMake
-if(TARGET some_target)
-    # True if some_target is an existing target
-endif()
+foreach(loop_var RANGE list_length)
+    # loop_var starts with 0 and goes up to (and inlcuding) list_length
+endforeach()
 ```
-
-```CMake
-if(DEFINED something_defined)
-    # True if something_defined is defined
-endif()
-```
-
 ```CMake
-if(EXISTS some_file_or_directory)
-    # True if some_file_or_directory is an existing file or directory
-endif()
+foreach(loop_var start stop [step])
+    # loop_var starts with the value 'start' and goes up to (and inlcuding) 'stop'
+    # with a increment of 'step'
+endforeach()
 ```
+foreach(loop_var ZIP_LISTS list_1 [list_n])
+    # you get a 'loop_var' for each list witch contains the value of in that list
+endforeach()
```

Comment:
```suggestion
```CMake
foreach(loop_var ZIP_LISTS list_1 [list_n])
    # you get a 'loop_var' for each list witch contains the value of in that list
endforeach()
```
```